resources/web/wwi/Parser.js

```diff
@@ -633,7 +635,9 @@ export default class Parser {
 
     const id = this.#parseId(node);
 
+    const type = getNodeAttribute(node, 'type', '').toLowerCase();
     const translation = convertStringToVec3(getNodeAttribute(node, 'translation', '0 0 0'));
+    const scale = convertStringToVec3(getNodeAttribute(node, 'scale', '1 1 1'));
```

Comment:
That seems unrelated to this PR, is it fixing a bug?

Reply:
No, it is a merge issue.
This lines have been removed from the develop branch in the Pose PR.